src/pages/about.js

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react'
-import { Link } from 'gatsby'
+import { PrismicLink } from '@prismicio/react'
```

Comment:
please delete this file also: https://prismic.io/docs/technologies/tutorial-4-add-dynamic-pages-gatsby#1.-delete-the-hardcoded-pages